Gruntfile.js

```diff
@@ -605,9 +626,10 @@ module.exports = function(grunt) {
     ]
   );
 
-  grunt.registerTask('jade_compile', ['jade', 'notify:jade_compile']);
+  grunt.registerTask('pug_compile', ['pug', 'notify:pug_compile']);
```

Comment:
We will see people poppin' around saying there automation doesn't work anymore because of this
Maybe we should let an alias `jade_compile` -> `pug_compile` ?